watson_developer_cloud/speech_to_text_v1.py

```diff
@@ -55,6 +56,10 @@ def recognize(self, audio, content_type, continuous=None, model=None,
                   'smart_formatting': smart_formatting,
                   'speaker_labels': speaker_labels}
 
+        if (params['customization_id'] is not None and 
+            customization_weight is not None):
+            params['customization_weight'] = customization_weight
+
```

Comment:
I think you can simply add
```
    'customization_weight': customization_weight,
```
into the params object above.  The check for `customization_weight is not None` isn't needed because the service will strip it out, and the check for 'customization_id' _shouldn't_ be needed because the service should simply ignore it when there is no customization_id.

Reply:
Ah, touche. I wasn't counting on the service to do that 😛 , but that's good to know !